packages/components/src/core/Dropdown/index.tsx

```diff
@@ -159,7 +159,7 @@ const Dropdown = <
             {buttonPosition === "left" ? (
               <div>
                 <StyledButton
-                  onClick={handleClose}
+                  onClick={(event) => handleClose(event, "selectOption")}
```

Comment:
maybe worth keeping the code DRY by creating a handler function 😄 ?

E.g.,

```ts
function handleButtonClose(event) {
  handle(event, "selectOption");
}
```

Reply:
Oh, for sure! Nice catch 😯